README.md

```diff
@@ -114,3 +114,66 @@ If you have a previous Percy configuration file, migrate it to the newest versio
 ```sh-session
 $ percy config:migrate
 ```
+
+## Running Percy on Automate
```

Comment:
```suggestion
#### Running Percy on Automate
```

Reply:
similar for other repos

Reply:
Issue is this is a different command and what I see is that migration config or installation of cli they are at `####` or `###` hence changing this to `####` will be weird.

Reply:
ack, btw we're planning to move stuff to `docs.percy.io` to remove redundancy. though its fine for now.